### PR TITLE
[11.0][MIG] ao_mail

### PR DESCRIPTION
**Pull request by MiquelRForgeFlow**
Migration scripts will be needed.

### CODE REVIEW

**Comment by LoisRForgeFlow** [comment]
In this module is really important to test the effect of the template sending emails. Please, can you check that and provide a screenshot?

File: ao_mail/models/res_partner.py [hunk 1]
```diff
@@ -1,40 +1,24 @@
...
 
-from odoo import api, fields, models
+from odoo import api, models
 
 
-class ResPartner(models.Model):
+class Partner(models.Model):
     _inherit = 'res.partner'
 
-    notify_email = fields.Selection(
```
Comment (LoisRForgeFlow) [inline ao_mail/models/res_partner.py]: this is migrated to field notification_type in user, no need to create a boolean in user.

**Review by LoisRForgeFlow — APPROVED** [review]
Some fixes. Migration script tested and :+1:

File: ao_mail/models/res_partner.py [hunk 2]
```diff
@@ -15,7 +13,7 @@ def _notify(self, message, force_send=False,
         if message.message_type == 'notification':
             notif_partners = self.filtered(
                 lambda partner: partner.mapped(
-                    'user_ids.email_notifications'))
+                    'user_ids.notifications_by_email'))
```
Comment (MiquelRForgeFlow) [inline ao_mail/models/res_partner.py:16]: -> `notification_by_email`

Comment (MiquelRForgeFlow) [reply ao_mail/models/res_partner.py:16]: Fixed in https://github.com/Eficent/ao-odoo/pull/59.

Comment (LoisRForgeFlow) [reply ao_mail/models/res_partner.py:16]: Solved in other PR. Thanks

**Pull request merged**